base_link: 'base'

end_effector_name: 'crustcrawler_gripper'

# Default grasp params
joints: ['right_finger_joint', 'left_finger_joint']

pregrasp_posture: [0.84, -0.84]
pregrasp_time_from_start: 4.0

grasp_posture: [0.0, 0.0]
grasp_time_from_start: 4.0

postplace_time_from_start: 4.0

# Desired pose from end effector to grasp [x, y, z] + [R, P, Y]
grasp_pose_to_eef: [0.0, 0.0, 0.0]
grasp_pose_to_eef_rotation: [0.0, 0.0, 0.0]

end_effector_parent_link: 'link_gripper_base'

approach_retreat_desired_dist: 0.1
approach_retreat_min_dist: 0.05
grasp_depth: 0.12
object_size: 0.04
angle_resolution: 16
ee_parent_link: 'link_gripper_base'
ee_group: gripper